src/components/container/PartnerBanner/PartnerBanner.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './PartnerBanner.scss';
+import Logo from './assets/partnerLogos/prgh-dark-main.svg';
+//this method gonna leads to so many imports so this is the right method?
+
+function PartnerBanner() {
+  const BANNERS = Array(10).fill({ name: "PRGH", src: Logo }); //dev version to add multiple logos
+
+  return (
+    <div className='PartnerBanner'>
+      <div>
+        <img alt="start" />
+        {BANNERS.map((b, i) => <img key={i} src={b.src} alt={b.name} />)}

```

Comment:
formatting

Reply:
<img alt="start" />
this is something I put manually to see the starting imaging and ending image to check every image is looping.
if you meant the indent in line 13